File: 2026/a.py
#!/usr/bin/env python3

# ICPC Asia Japan 2026 Online First-Round Contest
# Problem A: Find the strongest card

while True:
    n = int(input())
    if n == 0:
        break
    cs = list(map(int, input().split()))
    assert len(cs) == n
    m = max(cs, key=lambda c: (c + 13) % 16)
    print(m)

File: 2026/a.scala
#!/usr/bin/env scala

// ICPC Asia Japan 2026 Online First-Round Contest
// Problem A: Find the strongest card

import scala.io.StdIn

def main(): Unit = {
    val n = StdIn.readInt()
    if (n != 0) {
      val cs: Array[Int] = StdIn.readLine().split(' ').map(_.toInt)
      assert(cs.length == n)

      val m = cs.maxBy(c => (c + 10) % 13)
      println(m)

      main()
    }
}

main()
